src/sentry/static/sentry/app/views/releasesV2/list/index.tsx

```diff
@@ -145,7 +146,17 @@ class ReleasesList extends AsyncView<Props, State> {
       );
     }
 
-    if (this.getSort() !== 'date') {
+    if (activeSort === 'users_24h') {
+      return (
+        <EmptyStateWarning small>
+          {t(
+            'There are no releases with active users data (users in the last 24 hours).'
```

Comment:
```suggestion
            'There are no releases with active user data (users in the last 24 hours).'
```

Another option could be 

```suggestion
            'There are no releases with active users (users in the last 24 hours).'
```

Reply:
Thank you.